src/components/EventMapMarker.css

```diff
@@ -1,9 +1,47 @@
 .EventMapMarker-dot {
-  width: 12px;
-  height: 12px;
-  background-color: #f243a5;
-  border-radius: 15px;
-  border: 4px solid rgba(255, 255, 255, 0.7);
+  width: 13px;
+  height: 13px;
+  border-radius: 30px;
   cursor: pointer;
+  background: radial-gradient( farthest-side ellipse at 65% 35%, #fcb2c2, #ff0090);
   background-clip: padding-box;
+  -webkit-box-shadow:
```

Comment:
Like I told before. You do not need to have the prefix versions in the CSS. It will automatically add them.
So go ahead and remove all the prefix lines (`-webkit-`, `-moz-`) and just leave the `box-shadow` line in there. Makes the CSS much smaller and easier to read.